susemanager-utils/testing/automation/push-to-obs.sh

```diff
@@ -97,19 +97,24 @@ echo "End of task at ($(date). Logs for each package at ${GITROOT}/logs/"
 set +e
 
 PRET=0
+PKG_FAILED=""
 for i in $PIDS;do
-    echo "Waiting for pid $i"
+    package_name=$(basename $(readlink ${GITROOT}/logs/${i}.log) | cut -d"." -f1 )
+    echo "Waiting for process with pid $i, building package ${package_name}"
     wait $i
     result=$?
     echo "$i finished, result is $result"
-    cat ${GITROOT}/logs/${i}.log
     if [ $result -ne 0 ];then
-        echo "Seems there was an error with pid $i"
+        echo "Seems there was an error with process pid ${i}"
+        echo "When building package ${package_name}"
+        cat ${GITROOT}/logs/${i}.log
+        PKG_FAILED="${PKG_FAILED} ${package_name}"
         PRET=-1
     fi
 done
 
 if [ $PRET -ne 0 ];then
+    echo "$Fail to build packages ${PKG_FAILED}"
```

Comment:
Doesn't seem this `$Fail` environment exists, but anyway I suggest:

```suggestion
    echo "The following packages failed to build:"
    for p in ${PKG_FAILED}; do echo "- ${p}"; done
    echo "Please review the logs at ${GITROOT}/logs/"
```
Will be more readable if there are a few (or many) packages failing to build :-)

And since they can check the workspace it's easier to review file by file than looking at the log above.

If you like this approach, remember to remove the next line.